docs/introduction/composition.md

```diff
@@ -437,8 +433,6 @@ spec:
             endIpAddress: 10.10.255.254
             virtualNetworkSubnetIdSelector:
               name: sample-subnet
-        providerConfigRef:
-          name: example
         reclaimPolicy: Delete
```

Comment:
I believe this and other instances in this file can be removed or replaced with `deletionPolicy`